Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -131,7 +130,7 @@ - (void)setSelectionColor:(RCTUIColor *)selectionColor
   NSMutableDictionary *selectTextAttributes = self.selectedTextAttributes.mutableCopy;
   selectTextAttributes[NSBackgroundColorAttributeName] = selectionColor ?: [NSColor selectedControlColor];
   self.selectedTextAttributes = selectTextAttributes.copy;
-  self.insertionPointColor = self.selectionColor ?: [NSColor selectedControlColor];
+  self.insertionPointColor = self.selectionColor ?: [NSColor textColor];
```

Comment:
Feels like this was just a "we used the wrong NSColor when we made this" kind of problem?

Reply:
These are really subtle details and hard to notice. We found out when working on Messenger ;-)

Reply:
Yeah.. and the colors themselves aren't well documented for what their use case is 🤷🏾